AutomatedTesting/Gem/PythonTests/scripting/Node_HappyPath_DuplicateNode.py

```diff
@@ -34,80 +46,67 @@ def Node_HappyPath_DuplicateNode():
 
     :return: None
     """
-    from PySide2 import QtWidgets, QtTest
-    from PySide2.QtCore import Qt
-
-    from editor_python_test_tools.utils import Report
-    from editor_python_test_tools.utils import TestHelper as helper
-    import editor_python_test_tools.pyside_utils as pyside_utils
-
-    import azlmbr.legacy.general as general
-
-    WAIT_FRAMES = 200
 
-    NODE_NAME = "Print"
-    NODE_CATEGORY = "Debug"
-    EXPECTED_STRING = f"{NODE_NAME} - {NODE_CATEGORY} (2 Selected)"
 
-    def command_line_input(command_str):
-        cmd_action = pyside_utils.find_child_by_pattern(
-            sc_main, {"objectName": "action_ViewCommandLine", "type": QtWidgets.QAction}
+    def run_command_line_input(self, sc_editor, sc_editor_main_window, command_str):
+        command_line_action = pyside_utils.find_child_by_pattern(
+            sc_editor_main_window, {"objectName": "action_ViewCommandLine", "type": QtWidgets.QAction}
         )
-        cmd_action.trigger()
-        textbox = sc.findChild(QtWidgets.QLineEdit, "commandText")
+        command_line_action.trigger()
+        textbox = sc_editor.findChild(QtWidgets.QLineEdit, "commandText")
         QtTest.QTest.keyClicks(textbox, command_str)
         QtTest.QTest.keyClick(textbox, Qt.Key_Enter, Qt.NoModifier)
```

Comment:
Which command line is this? The editor console command line?

The name of the function and lack of comments or class context makes this nearly impossible to know what this is doing by just looking at  the code. Even the test description doesn't give context on why we're doing this and all I know from usage is that you're adding a node using this command.